pkg/ingester/ingester.go

```diff
@@ -641,7 +642,12 @@ func (i *Ingester) Push(ctx context.Context, req *logproto.PushRequest) (*logpro
 
 // GetStreamRates returns a response containing all streams and their current rate
 // TODO: It might be nice for this to be human readable, eventually: Sort output and return labels, too?
-func (i *Ingester) GetStreamRates(_ context.Context, _ *logproto.StreamRatesRequest) (*logproto.StreamRatesResponse, error) {
+func (i *Ingester) GetStreamRates(ctx context.Context, _ *logproto.StreamRatesRequest) (*logproto.StreamRatesResponse, error) {
+	if sp := opentracing.SpanFromContext(ctx); sp != nil {
+		sp.LogKV("event", "ingester started to handle get stream rates")
+		defer sp.LogKV("event", "ingester finished to handle get stream rates")
```

Comment:
```suggestion
		defer sp.LogKV("event", "ingester finished handling get stream rates")
```
Maybe you can use the full function name (GetStreamRates) instead of "get stream rates". That is clearer.